package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@getflywheel/local-components",
-	"version": "13.2.7",
+	"version": "13.2.8",
```

Comment:
No need to bump the version here! This will get taken care of in the deploy step. https://github.com/getflywheel/flywheel-local/wiki/Publishing-local-components

Reply:
Corrected, thanks for catching that.